### DISCUSSION TITLE
Has enableSmoothScroll been removed in v3.0.0-beta.49

### DISCUSSION THREAD

**Discussion by greg-wiltshire** [TanStack/virtual#514] [Q&A] [answered]
I was using the latest release (v3.0.0-beta.9) for a table with dynamic row heights and measureElement. It was working well and had no flicker when scrolling with enableSmoothScroll set to false.

I have updated to v3.0.0-beta.49 and it looks like enableSmoothScroll is no longer available, is this correct or has it been moved to a different option/property?

**Comment by piecyk** [comment, marked as answer]
Yes, enableSmoothScroll was remove, as the smooth scroll is not enabled by default. Overall it's not fully supported in dynamic case https://github.com/TanStack/virtual/discussions/495#discussioncomment-4886406
